### PR TITLE
Remove toggling services

### PR DESCRIPTION
**Pull request by lvillen**
**What this PR does / why we need it**:

This PR deletes some old code which doesn't relate on how we work with Products nowadays. 

In the past we had multi-service accounts (see issue[ #5927](https://github.com/3scale/system/issues/5927) on System repo) and the solution we came up with was a toggling system for Services Index and Dashboard (see System PRs [#5927](https://github.com/3scale/system/issues/5927), [#6264](https://github.com/3scale/system/pull/6271) and [#6366](https://github.com/3scale/system/pull/6366)). 

This is not the case anymore so we can move forward without this code.

### CODE REVIEW

File: app/views/api/services/show.html.slim [hunk 1]
```diff
@@ -1,6 +1,3 @@
-- content_for :javascripts do
```
Comment (mayorova) [inline app/views/api/services/show.html.slim:1]: There is a block at the end of this file that should be removed too:
```
javascript:
  document.addEventListener('DOMContentLoaded', function () {
    serviceInitialize()
  })
```

**Review by mayorova — CHANGES_REQUESTED** [review]
Just one comment - otherwise looks good

**Review by mayorova — DISMISSED** [review]

**Review by thalesmiguel — APPROVED** [review]

**Review by josemigallas — CHANGES_REQUESTED** [review]
The following files still have traces:
- app/assets/stylesheets/provider/_services.scss (title-toggle, is-closed)
- app/javascript/src/Types/globals.d.ts (serviceInitialize)

**Review by josemigallas — APPROVED** [review]
🥇

**Pull request merged**